core/chain-configs/src/client_config.rs

```diff
@@ -286,6 +287,7 @@ pub fn default_view_client_throttle_period() -> Duration {
     Duration::seconds(30)
 }
 
+/// Returns the default value for maximum size of state witnesses in the OrphanStateWitnessPool.
 pub fn default_trie_viewer_state_size_limit() -> Option<u64> {
```

Comment:
This comment should be on the `default_orphan_state_witness_max_size()` function

Reply:
My bad, moved the comment, added another comment here and edited this one.